Screenbox/Pages/SettingsPage.xaml

```diff
@@ -113,6 +113,7 @@
                 Padding="{StaticResource PageContentMargin}">
                 <StackPanel.ChildrenTransitions>
                     <RepositionThemeTransition IsStaggeringEnabled="False" />
+                    <EntranceThemeTransition IsStaggeringEnabled="True" />
```

Comment:
```suggestion
                    <EntranceThemeTransition FromVerticalOffset="50" IsStaggeringEnabled="True" />
```

Reply:
Is there any particular reason for this offset value?

Reply:
No particular reason, there seems to be a consensus on that value:
[1](https://github.com/CommunityToolkit/Windows/blob/main/components/SettingsControls/samples/SettingsPageExample.xaml#L32) [2](https://github.com/microsoft/calculator/blob/main/src/Calculator/Views/Settings.xaml#L80)